### DISCUSSION TITLE
Cannot click the toolbars inside the modal

### DISCUSSION THREAD

**Discussion by yoouTaro** [igordanchenko/yet-another-react-lightbox#284] [Q&A] [answered]
I'm currently using the Dialog component from [Shadcn UI](https://ui.shadcn.com/docs/components/dialog), and I'm experiencing an issue where I cannot click on any toolbar elements within the dialog.

**Comment by igordanchenko** [comment]
Please share a repro. You can use the following sandbox as a template.

https://stackblitz.com/edit/yet-another-react-lightbox-sandbox?file=src%2FApp.tsx

**Reply by yoouTaro** [reply]
Hi,

Thank you for your response. I’ve created a sandbox for you. You can access it here:

https://stackblitz.com/edit/yet-another-react-lightbox-sandbox-ium98b?file=src%2FApp.tsx

**Reply by igordanchenko** [reply, marked as answer]
Thank you very much for providing the sandbox. The behavior you see is happening due to Radix UI Dialog adding `pointer-events: none` style to the document `<body>`:

<img width="515" alt="Screenshot 2024-06-18 at 10 11 46 PM" src="https://github.com/igordanchenko/yet-another-react-lightbox/assets/64441155/4f8d9e71-ac49-4c4a-a362-c59731195699">

Here is the workaround you can use to counter this behavior:

```jsx
<Lightbox
  styles={{ root: { pointerEvents: "auto" } }}
  // ....
/>
```

**Reply by yoouTaro** [reply]
Thank you for your help! The solution you provided fixed my issue.
